resources/views/instructors/create.blade.php

```diff
@@ -0,0 +1,159 @@
+                                <div class="control is-expanded">
+                                    <input class="input" type="text" id="Address1" name="Address1" placeholder="Address Line 1" required/>
+                                </div>
+                            </div>
+                            <div class="field">
+                                <div class="control is-expanded">
+                                    <input class="input" type="text" id="Address2" name="Address2" placeholder="Address Line 2" />
+                                </div>
+                            </div>
+                            <div class="field">
+                                <div class="control is-expanded">
+                                    <input class="input" type="text" id="Address3" name="Address3" placeholder="Address Line 3" />
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="field is-horizontal">
+                        <div class="field-label is-normal">
+                            <label class="label">City, State and Zip</label>
+                        </div>
+                        <div class="field-body">
+                            <div class="field">
+                                <div class="control is-expanded">
+                                    <input class="input" type="text" id="City" name="City" placeholder="City" required/>
+                                </div>
+                            </div>
+                            <div class="field">
+                                <div class="control is-expanded">
+                                    <input class="input" type="text" id="State" name="State" placeholder="State" required/>
+                                </div>
+                            </div>
+                            <div class="field">
+                                <div class="control is-expanded">
+                                    <input class="input" type="text" id="Zip" name="Zip" placeholder="Zip" required/>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                @else
+                    @foreach ($instructor->addresses as $address)
+                        <div class="field is-horizontal">
+                            <div class="field-label is-normal">
+                                <label class="label">Address {{$address->id}}</label>
+                            </div>
+                            <div class="field-body">
+                                <div class="field">
+                                    <div class="control is-expanded">
+                                        <input class="input" type="text" id="Address1" name="Address1" placeholder="Address Line 1" value="{{$address->Address1}}" required/>
```

Comment:
Usually for a create view, there wouldn't be any values set in these fields. Also an ID doesn't need to be specified for any form input